src/config/index.js

```diff
@@ -42,13 +99,39 @@ class Config {
     this.removeAllScriptNames();
     this.removeAllScripts();
 
-    if (fse.existsSync(CONFIG_FILE_PATH)) {
-      const { shell, scriptNames } = getConfig();
+    if (fse.pathExistsSync(CONFIG_FILE_PATH)) {
+      const { shell, scriptNames } = loadConfigJson();
       this.updateShell(shell);
       scriptNames.forEach(scriptName => this.addScriptName(scriptName));
       scriptNames
-        .map(scriptName => `${SCRIPTS_DIRECTORY_PATH}/${scriptName}.yml`)
-        .map(scriptName => Parser.getScript(scriptName))
+        // Map saved script name to yaml or json file path
+        .map(scriptName => {
+          let fileName = '';
+          ['.yml', '.simple.yml', '.json', '.simple.json'].forEach(possibleExtension => {
+            const test = `${SCRIPTS_DIRECTORY_PATH}/${scriptName}${possibleExtension}`;
+            if (fse.pathExistsSync(test)) {
+              fileName = test;
+            }
+          });
+          if (isEmptyString(fileName)) {
+            this.removeScriptName(scriptName);
+          }
+          return fileName;
+        })
+        .filter(fileName => !isEmptyString(fileName))
+        .map(fileName => {
+          const scriptType = isSimpleScript(fileName) ? ScriptTypes.SIMPLE : ScriptTypes.ADVANCED;
```

Comment:
nit:
- [ ] refactor this ternary statement into a `getScriptType` utility function that returns the `scriptType` and replace usages